pkg/sql/importer/import_job.go

```diff
@@ -1462,31 +1461,33 @@ func (r *importResumer) dropTables(
 		return nil
 	}
 
-	var revert []catalog.TableDescriptor
-	var empty []catalog.TableDescriptor
+	var empty bool
```

Comment:
nit: rename variable to `tableWasEmpty` or something more self explanatory

Reply:
Done.